src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -185,4 +200,12 @@ private JsonObject customContentTab(String tabType, String name, JsonObject prop
         result.add("props", props);
         return result;
     }
+
+    private boolean isSmartExperiment(ImmutableList<String> technologyType) {
+        Stream<Boolean> isSmartExperiment = technologyType.stream()
+                .map(type -> type
+                        .toLowerCase()
+                        .matches(SMART_TECHNOLOGY_TYPE + "-(?:.*)"));
+        return isSmartExperiment.anyMatch(technology -> technology);
```

Comment:
Be careful as all this part was reviewed and merged before in 5f9a40840bb77ca8cd3fd32f716a1c07fefbed7d and ended up being quite different.

Reply:
Thanks for this helpful reminder. I will rebase the master branch into this PR before merging.

Reply:
Thanks for this helpful reminder. I will resolve the conflicts carefully before merging.